sdk/core/core-client-rest/src/apiVersionPolicy.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PipelinePolicy } from "@azure/core-rest-pipeline";
+import { ClientOptions } from "./common";
+import { URL } from "./url";
+
+export const apiVersionPolicyName = "ApiVersionPolicy";
+
+/**
+ * Creates a policy that sets the apiVersion as a query parameter on every request
+ * @param options - Client options
+ * @returns Pipeline policy that sets the apiVersion as a query parameter on every request
+ */
+export function getApiVersionPolicy(options: ClientOptions): PipelinePolicy {
```

Comment:
There's not usually a prefix in the factory function name. At first I thought this policy "got" the api version somehow rather than setting it 😆 

```suggestion
export function apiVersionPolicy(options: ClientOptions): PipelinePolicy {
```

Reply:
Yeah good point!